qa/tasks/cephfs/test_volumes.py

```diff
@@ -5842,6 +5858,9 @@ def test_subvolume_snapshot_info_if_clone_pending_for_no_group(self):
         for clone in clone_list:
             self._fs_cmd("subvolume", "rm", self.volname, clone)
 
+        # set the config options back to default
+        self._set_config_options_back_to_default()
```

Comment:
Comment `# set the config options back to default` is redundant, method name is sufficient.